### PR TITLE
Liquid and transparent

### PR DESCRIPTION
**Pull request by sjohnston92**
Issue:

### CODE REVIEW

**Comment by TomWBush** [comment]
Have you tested it? It's not working in the preview above.

File: src/components/animated/_liquid.less [hunk 1]
```diff
@@ -37,12 +44,12 @@
     color: #000 !important;
     .bc(@color);
 
-    &:hover::before {
-      top: -120px;
+     &.animated:before {
```
Comment (shahednasser) [inline src/components/animated/_liquid.less:47]: Why did you remove `&:hover` ? it should be for both. Something like this:

```
&:hover:before, &.animated:before
```
I also think you should combine after with it as well since it's the same rule. It'll be something like this:

```
&:hover:before, &.animated:before, &:hover:after, &.animated:after
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
It seems like you didn't run `npm run compile` and commit the changes so you need to run that for changes to be seen.

I also still need you to remove the changes in `package-lock.json` if you're having trouble with that you can just copy the content of the file from this master repository.

Keep in mind that when you need to make any changes you don't need to create a new PR each time. Just commit the changes and they will be seen here!

**Comment by shahednasser** [comment]
@sjohnston92 I saw you closed this. Will you still be working on these buttons?

**Comment by sjohnston92** [comment]
@shahednasser yes

**Comment by sjohnston92** [comment]
@shahednasser yes still will be working on them, will be submitting the revision later today
